### ISSUE TITLE
COSIMA workshop 2023 poster

### ISSUE THREAD

**Issue by wghuneke** [willaguiar/DSW-collaborative-project#23]
Hi team, I've put together a poster for the upcoming COSIMA workshop. let me know if you want me to make any changes!

**Comment by wghuneke** [comment]
[COSIMA_2023_Poster_DSW_ACCESS-OM2-01.pdf](https://github.com/willaguiar/DSW-collaborative-project/files/12460616/COSIMA_2023_Poster_DSW_ACCESS-OM2-01.pdf)

**Comment by dkhutch** [comment]
Hi Wilma,
Nice poster - clear, simple layout, good for creating discussion. I would try to add somewhere prominently that "No fully coupled climate models have ever produced realistic DSW".

Then later on, I would want to emphasise (bold text?) that adding the CM2 wind field has *surprisingly* enhanced and densified the DSW, which suggests that if we could replicate this ocean-sea ice configuration into a coupled model, it may be able to generate DSW in fully coupled mode (first time ever)... but we need to test it. 

Perhaps I'm over-selling it? That to me seems like an exciting result to highlight some more.
Cheers, Dave

**Comment by PaulSpence** [comment]
Thanks Wilma :) Looks good enough to me. Looking forward to chatting with you all about it next week :)

**Comment by willaguiar** [comment]
Awesome Wilma, Looks good to me :)

**Comment by adele157** [comment]
Thanks for putting this together Wilma! My suggestions:

- Don’t use the DSW acronym in the title. I would estimate 50% of COSIMA
won’t know what that means. “Antarctic dense water” instead? The acronym
also needs defining in the purple box.

- if it was me I would put Wilton’s name first, with presenter name after
in bold, to give him a bit more credit for all the effort he’s put into
getting these runs going. But then again, we did say at the start that we
would leave author order open and see who does the most. And it is nice in
a collaborative project for us all to be able to feel ownership of the
project. So happy either way!

- Under “modelling challenges”, I suggest “other GLOBAL eddying models fail
to reproduce…” because other regional models do get it (eg Mensah et al.
2021).

- under “sensitivity experiments “, “increase surface grid to 5m” is not
what we did right? Wilton is running that simulation now, but I think the
simulation you’re plotting has coarsened resolution over the top 500m. Does
that run still have 75 levels? I can’t remember.

- do the SWMT and export plots show the same information? If yes, then
maybe cut the SWMT plots to simplify it. You could also then zoom in the
axes a bit on the export plots so it’s easier to see the changes. As it is,
it’s hard to tell that the different simulations are very different. Could
show something different instead, like changes in bottom age? That would
show which regions are getting more / less DSW export.

- I like the link to the GitHub repo!

**Comment by wghuneke** [comment]
Thanks to all for the feedback, I'll make some adjustments!

**Comment by willaguiar** [comment]
ok, answering Adele, the run with coarsened vertical resolution at the top 500m has 60 levels, and the new run with changes only in the top 5 m has 72 levels.

**Comment by wghuneke** [comment]
Quick summary from the poster session:

- Lots of interest in general
- Vertical grid: It is NOT the uppermost grid cell that matters, but the resolution below (the mixed layer) → Wilton finished the run where only the uppermost grid cell is coarsened and it shows similar results to control
- CM2 winds (but also vertical grid): largest anomalies occur in the Weddell Sea, we should focus our analysis on the region (sea ice, wind stress)
- Suggestions: 
> - run control and get 3hr output for sea ice fluxes, force coarsened grid with that to distinguish between sea ice feedback and grid
> - look at mixing

**Comment by willaguiar** [comment]
Thanks Wilma! I just need to correct something I said before. Upon closer look, it seems that the simulation with just the top cell change (5mtop) do have the same SWMT pattern and transport across the 1000m isobath ( I had some errors in the post before - sorry). But at least this points out that the topmost 5m cell is what actually matters.

I also finished running a simulation with stronger restoring parameters (max_delta_salinity_restore = 5), and we still have very similar SWMT and volume transports as the ctrl case, so Salinity restoring seem to not be doing much here.

**Comment by wghuneke** [comment]
When you say "the simulation with just the top cell change (5mtop) do have the same SWMT pattern and transport across the 1000m isobath", you mean compared to control or the previous coarsened grid experiment? If it's the upper grid cell, that would be neat (at least one experiment that fits our hypothesis).

Nice result that surface salinity restoring does not seem to have an impact. Freshwater fluxes controlled by sea ice might still be important though?

**Comment by willaguiar** [comment]
Yes -  the 5mtop has the same result as the coarsened vertical resolution case ( same reduction on both SWMT and cross-slope mass transport).

**Comment by adele157** [comment]
Excellent, that's a nice result then! I wonder if we should test this
further?

e.g. run a case with a thinner upper layer, perhaps halving the upper layer
compared to the control.

Could we also test that the dependence on thickness of the upper cell comes
down to the sea ice water/salt fluxes being extracted from the upper cell?
e.g. if we could reduce the upper layer thickness to 0.5m, but still
extract/insert water fluxes from the top 2 layers (1m). This could be
technically challenging though.
